preprocessing/markdown_preprocessing/processMarkdownData.py

```diff
@@ -189,8 +191,9 @@ def get_data_from_h2_header(self, article: Article,
 
                 articleSection.ner_tuples[bullet_point_index] = []
                 for href_words in h2_bullet_point.findAll(href=True):
-                    word, href_link = href_words.text, href_words['href'].split(
-                        'https://ref.ly/logos4/Factbook?ref=')[-1]
+                    word, href_link = href_words.text, href_words[
+                        'href'].split(
```

Comment:
Oh you can ignore this 

Its because I'm using a linter (yapf) that activates when I autosave and it has a line limiti + formats the code nicer

Its just formatting thats all